src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/removal-under-maintenance/RemovalUnderMaintenanceView.tsx

```diff
@@ -0,0 +1,219 @@
+import { useL10n } from "../../../../../../../../../hooks/l10n";
+import { TelemetryLink } from "../../../../../../../../../components/client/TelemetryLink";
+import { TelemetryButton } from "../../../../../../../../../components/client/TelemetryButton";
+import { CONST_URL_SUMO_MANUAL_REMOVAL } from "../../../../../../../../../../constants";
+import { useState } from "react";
+
+export type Props = {
+  data: OnerepScanResultRow[];
+  stepDeterminationData: StepDeterminationData;
+  subscriberEmails: string[];
+};
+
+export const RemovalUnderMaintenanceView = (props: Props) => {
+  if (props.data === null) {
+    console.error("No scan data");
+  }
+  const l10n = useL10n();
+  const testScanItem = props.data;
+  const firstScan = testScanItem[0];
+
+  const [detailedRemovalGuide, setDetailedRemovalGuide] = useState(false);
+
+  const dataBrokerInformation = (
+    <div className={styles.dataBrokerInformationWrapper}>
+      <p className={styles.header}>
+        {l10n.getFragment("data-broker-removal-maintenance-header", {
+          elems: {
+            link_to_data_broker: (
+              <TelemetryLink
+                href={firstScan.link}
+                target="_blank"
+                eventData={{
+                  link_id: `go_to_${firstScan.data_broker}_link`,
+                }}
+              />
+            ),
+          },
+          vars: {
+            data_broker_name: firstScan.data_broker,
+          },
+        })}
+      </p>
+      <div className={styles.exposureCardWrapper}>
+        <DataBrokerRemovalMaintenance
+          scanResult={testScanItem[0]}
+          isPremiumUser={true}
+        />
+
+        <div className={styles.removalContentSection}>
+          <dt>
+            {l10n.getString(
+              "data-broker-removal-maintenance-steps-to-remove-header",
+            )}
+          </dt>
+          <dd>
+            <ol>
+              <li>
+                {l10n.getString(
+                  "data-broker-removal-maintenance-steps-to-remove-header-step-one",
+                )}
+              </li>
+              <li>
+                {l10n.getString(
+                  "data-broker-removal-maintenance-steps-to-remove-header-step-two",
+                )}
+              </li>
+            </ol>
+          </dd>
+          <TelemetryButton
+            variant="link"
+            onPress={() => setDetailedRemovalGuide(true)}
+            event={{
+              module: "button",
+              name: "click",
+              data: {
+                button_id: "view_removal_instructions",
+              },
+            }}
+          >
+            {l10n.getString(
+              "data-broker-removal-maintenance-steps-to-remove-view-more-link",
+            )}
+          </TelemetryButton>
+        </div>
+
+        <div className={styles.removalContentSection}>
+          <dt>
+            {l10n.getString(
+              "data-broker-removal-maintenance-steps-to-remove-header",
```

Comment:
This header would need to be:

```suggestion
              "data-broker-removal-maintenance-rationale-header",
```